library_management_cookiecutter/library/mixin.py

```diff
@@ -0,0 +1,26 @@
+from django.contrib.auth.mixins import PermissionRequiredMixin
+from django.shortcuts import redirect
+from django.contrib import messages
+
+
+class MyCustomPermissions(PermissionRequiredMixin):
+    
+    def has_permission(self):
+        user_permissions = self.request.user.get_all_permissions()
+        # print(self.permission_required)
+        print(self.request.method)
```

Comment:
remove print statement 
suggestion. Use pre commit. Understand it What is pre commit  and create a hook for black and removeprint